code/modules/food_and_drinks/food/snacks_pastry.dm

```diff
@@ -261,6 +270,16 @@
 	filling_color = "#879630"
 	value = FOOD_EXOTIC
 
+/obj/item/reagent_containers/food/snacks/donut/jelly/oreo
+	name = "oreo jelly donut"
+	desc = "Goes great with a glass of moth milk."
```

Comment:
reusing descriptions smh

Reply:
and *this* one even

Reply:
it appears a lot of the donuts have the same descriptions. Uh?